test/CAS/mccas.swift

```diff
@@ -0,0 +1,26 @@
+// REQUIRES: OS=macosx && CPU=arm64
```

Comment:
why `CPU=arm64`? This test should work on x86 right? Using arm64 will get zero coverage on CI.

I will just only run on macOS for now and remove `-target` flag from the tests (`%target-swift-frontend` already builds for host triple)